functions/__sf_section_docker.fish

```diff
@@ -0,0 +1,51 @@
+#
+# Docker
+#
+# Current Docker version and Machine name.
+
+function __sf_section_docker -d "Display docker version and machine name"
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_DOCKER_SHOW true
+    __sf_util_set_default SPACEFISH_DOCKER_PREFIX "is "
+    __sf_util_set_default SPACEFISH_DOCKER_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+    __sf_util_set_default SPACEFISH_DOCKER_SYMBOL "🐳 "
+    __sf_util_set_default SPACEFISH_DOCKER_COLOR cyan
+    __sf_util_set_default SPACEFISH_DOCKER_VERBOSE_VERSION false
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_DOCKER_SHOW = false ]; and return
+
+    # Show docker version only when pwd has dockerfile or docker-compose.yml or COMPOSE_FILE
+    [ -f ./Dockerfile -o -f ./docker-compose.yml ]; or test "$COMPOSE_FILE"; or return
+
+    # Show Docker version only if docker is installed
+    type -q docker; or return
+
+    # Show Docker version only if docker deamon is up
+    docker ps > /dev/null 2>&1
```

Comment:
Do we need this `docker ps` call? The `docker version` call beneath should be sufficient, throwing an error if the daemon isn't up.